antlr/src/main/antlr/FSH.g4

```diff
@@ -5,7 +5,7 @@ options { tokenVocab = FSHLexer; }
 doc:                entity* EOF;
 entity:             alias | profile | extension | invariant | instance | valueSet | codeSystem | ruleSet | paramRuleSet | mapping | logical | resource;
 
-alias:              KW_ALIAS SEQUENCE EQUAL SEQUENCE;
+alias:              KW_ALIAS ALIAS_SEQUENCE ALIAS_EQUAL ALIAS_SEQUENCE NEWLINE;
```

Comment:
This requirement for a newline between Aliases is new, right? Is there a reason for introducing this new requirement here? Is it just so we can understand properly when to leave the new lexical mode?

Reply:
Yeah I couldn't figure out another way to know when to exit the lexical mode.

Reply:
No longer relevant as of https://github.com/FHIR/sushi/pull/880/commits/c2b7e1413b6cfc3b0096eb0a61dfdfc6f670e421.